### PR TITLE
Feat: Added Sorting by price feature

### PR DESCRIPTION
**Pull request by deepak29-git**
1,High to Low feat
2.Low to high feat

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import { Products } from "./Pages/Product/Products";
+import {Routes,Route} from 'react-router-dom'
+import {Home} from './Pages/Home/Home'
+import {WishList} from './Pages/WishList/WishList'
+import {Login} from './Pages/Login/Login'
+import {Signup} from './Pages/Signup/Signup'
+
+function App() {
+  return (
+    <div className="App">
```
Comment (spraveenofficial) [inline src/App.js:11]: You can remove this div.

File: src/Context/filter-context.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useReducer } from "react";
+import { reducer } from "../Reducer/reducer";
+import { useAxios } from "../Api-data/useAxios";
```
Comment (spraveenofficial) [inline src/Context/filter-context.js:3]: Try to keep better name useAxios is not reliable.

File: src/Pages/Home/Home.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import {Header} from "../../components/Header/Header"
+import {Category} from "../../components/Category/Category"
+import {Baner} from "../../components/Baner/Baner"
+import {Footer} from "../../components/Footer/Footer"
+
+const Home=()=>{
+    return(
+    <div>
```
Comment (spraveenofficial) [inline src/Pages/Home/Home.js:8]: Instead of this div use can use fragments.

```
<>
</>
```

File: src/Pages/Login/Login.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    display: flex;
+    justify-content: center;
+  }
+  
+  .input-group {
+    width: 380px;
+  }
+  
+  .right-arrow {
+    height: 20px;
```
Comment (spraveenofficial) [inline src/Pages/Login/Login.css:34]: Try to not use the px in css. instead to px you can move to rem or em.

File: src/Pages/Login/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import "../Login/Login.css";
+import { Link } from "react-router-dom";
+import {Header} from '../../components/Header/Header'
+export const Login = () => {
+  return (
+    <>
+    <Header/>
```
Comment (spraveenofficial) [inline src/Pages/Login/Login.jsx:7]: You can move this Header to your App.js directly.

File: src/backend/db/categories.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
+import { v4 as uuid } from "uuid";
+
+/**
+ * Category Database can be added here.
+ * You can add category of your wish with different attributes
+ * */
+
+export const categories = [
+  {
+    _id: uuid(),
```
Comment (spraveenofficial) [inline src/backend/db/categories.js:10]: Use Static id, it will work finer.

File: src/Context/filter-context.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+const FilterContext = createContext(null);
+
+const FilterProvider = ({ children }) => {
+  const { data } = useAxios();
+
+  console.log(data);
```
Comment (spraveenofficial) [inline src/Context/filter-context.js:10]: Remove console.log before pushing.

File: src/Pages/Login/Login.css [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    background-color: var(--primary-hover);
+  }
+  
+  .center {
+    display: flex;
+    justify-content: center;
+  }
+  
+  .input-group {
+    width: 380px;
```
Comment (spraveenofficial) [inline src/Pages/Login/Login.css:30]: You can avoid this much  static values.

**Review by spraveenofficial — COMMENTED** [review]
You can work on my suggestions given below.

File: src/Api-data/useAxios.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+import axios from "axios";
+const api = "/api/products";
+
+export const useAxios = () => {
+  const [data, setData] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try{
+        const {data} = await axios.get(api);
+        // console.log(data.products)
```
Comment (Mohitkumar33) [inline src/Api-data/useAxios.js:14]: Remove comment

**Review by Mohitkumar33 — APPROVED** [review]

**Pull request merged**